src/Host.h

```diff
@@ -143,7 +143,7 @@ class Host : public QObject
     int createStopWatch();
     void startSpeedWalk();
     void saveModules(int sync, bool backup = true);
-    void reloadModule(const QString& moduleName);
+    void reloadModule(const QString& reloadModuleName);
```

Comment:
TBH, the argument name is not a requirement in the header file - it is only useful IMO to disambiguate multiple arguments of the same type (or, of course, if it has a defaulted value where it is mandatory, isn't it?)...

Reply:
Yeah - the IDE did this